plugins/generic/pdfJsViewer/templates/display.tpl

```diff
@@ -74,9 +74,8 @@
 	<script type="text/javascript">
 		// Creating iframe's src in JS instead of Smarty so that EZProxy-using sites can find our domain in $pdfUrl and do their rewrites on it.
 		$(document).ready(function() {ldelim}
-			var urlBase = "{$pluginUrl}/pdf.js/web/viewer.html?file=";
-			var pdfUrl = {$downloadUrl|json_encode};
-			$("#pdfCanvasContainer > iframe").attr("src", urlBase + encodeURIComponent(pdfUrl));
+			var urlBase = '{$pluginUrl}/pdf.js/web/viewer.html?file=';
+			$("#pdfCanvasContainer > iframe").attr("src", urlBase + '{$downloadUrl}');
```

Comment:
Are you sure that the download URL doesn't need to be encoded?

Reply:
You are right. it should be encoded. 

Still, it seems I am having some issues with getting the download files. (possibly because of the introduction of the `files` table). There are places in the code that [https://github.com/pkp/omp/blob/master/plugins/generic/pdfJsViewer/PdfJsViewerPlugin.inc.php#L67](https://github.com/pkp/omp/blob/master/plugins/generic/pdfJsViewer/PdfJsViewerPlugin.inc.php#L67 ). Does `submissionFiles` have `mimetype` data now, or should we look at `files` service for it?

Reply:
The mimetype is part of the SubmissionFile data now. This was a recent change merged in just last week.